akka-docs/rst/scala/remoting-artery.rst

```diff
@@ -281,11 +281,56 @@ untrusted mode when incoming via the remoting layer:
   marking them :class:`PossiblyHarmful` so that a client cannot forge them.
 
 
-Lifecycle and Failure Recovery Model
-------------------------------------
-
-TODO
-
+Quarantine
+----------
+
+Akka remoting is using Aeron as underlying message transport. Aeron is using UDP and adds
+among other things reliable delivery and session semantics, very similar to TCP. This means that
+the order of the messages are preserved, which is needed for the :ref:`Actor message ordering guarantees <message-ordering>`.
+Under normal circumstances all messages will be delivered but there are cases when messages
+may not be delivered to the destination:
+
+* during a network partition and the Aeron session is broken
```

Comment:
mention that this can be automatically recovered by remoting once the parition is over.